docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,162 @@
+# Contributing policies
+
+This doc provides instruction on how to contribute policies to the `openshift-4-compliance-automation` repository.
+
+You can contribute policies by submitting a pull request (PR) in the `openshift-4-compliance-automation` repository. Your PR must be reviewed by the owners of the repo. The repository is welcoming every contributor to join the project and create an automated compliance implementation for OpenShift environments.
+
+## Guidelines
+
+The repository is based on 3 main tools -
+
+- Open Policy Agent Gatekeeper
+- Kyverno
+- Red Hat Advanced Cluster Management for Kubernetes
+
+Each tool will be associated with its own directory. 
+
+Each product's policies are devided into multiple compliance controls -
+
+- authentication-user-management
+- authorization
+- etcd-security 
+- infrastructure-general
+- monitoring-observability
+- networking 
+- resource-exhaustion
+- storage
+- trusted-image-sources
+
+Each compliance control will have its own directory in each tool directory. The compliance control directories are based own the next [doc](https://docs.google.com/spreadsheets/d/1vhtWJYgvbVlW58D-jL9s-iBA56aT3m9Sf_qnME9gDyw/edit?usp=sharing).
```

Comment:
<<
Each compliance control will have its own directory in each tool directory. The compliance control directories are based own the next [doc](https://docs.google.com/spreadsheets/d/1vhtWJYgvbVlW58D-jL9s-iBA56aT3m9Sf_qnME9gDyw/edit?usp=sharing).
---
Each compliance control will have its own directory in each tool directory. The compliance control directories are based on the next [doc](https://docs.google.com/spreadsheets/d/1vhtWJYgvbVlW58D-jL9s-iBA56aT3m9Sf_qnME9gDyw/edit?usp=sharing).
>>